freedomofpress/securedrop-proxy
# Uses new reproducible wheels from our builds

The new wheels are coming in the following PR:

https://github.com/freedomofpress/securedrop-debian-packaging/pull/213

These wheels are reproducible, you can build them following our
Makefile.

## How to test?

They sha256sums should match with the above mentioned PR.

https://github.com/freedomofpress/securedrop-debian-packaging/pull/213/files#diff-7899317ed7333cf6bdab16b9bf5773f780ecb55b11114fc715e9240b16fd76ff

There is a temporary commit, which we can remove before merge.
